acceptance/bundle/plan/no_upload/script

```diff
@@ -0,0 +1,9 @@
+trace $CLI bundle plan
+
+# Expect no clean up requests
+trace jq -s '.[] | select(.path=="/api/2.0/workspace/delete") | select(.body.path | test(".*/artifacts/.internal"))' out.requests.txt
```

Comment:
Why do we filter on /artifacts/.internal, are there other /delete/ requests? If yes, would be still good to see them with a comment that it's expected?

The concern there is that we only check certain paths but what we want is not remote changes at all.

Reply:
q - would it be helpful to just dump all non-GET requests?

Reply:
Good point, we might better check for any non GET requests indeed